syz-fuzzer/fuzzer.go

```diff
@@ -161,6 +162,8 @@ func main() {
 		faultInjectionEnabled = true
 	}
 
+	compsEnabled = checkCompsEnabled()
```

Comment:
There is another open of `/sys/kernel/debug/kcov` below to set a.Kcov. Make `checkCompsEnabled` return both flags, something along the lines of:
```
func checkCompsEnabled() (kcov, comps bool)
```
Probably need to rename the function somehow then as well.